lib/pages/home/modals/create_post/create_post.dart

```diff
@@ -182,7 +182,7 @@ class CreatePostModalState extends State<CreatePostModal> {
 
     return OBThemedNavigationBar(
       leading: GestureDetector(
-        child: const OBIcon(OBIcons.close),
+        child: const OBIcon(OBIcons.close, semanticLabel: 'Close create new post'),
```

Comment:
This should be localised right?

Reply:
Yes, but localising here might cause conflicts, remember we kinda decided to localise on develop or on a release branch for all PRs.